### DISCUSSION TITLE
module.css not applied outside folder app in NextJS 13

### DISCUSSION THREAD

**Discussion by nisamarwa** [vercel/next.js#59906] [Help]
### Summary

Hello, all. I'm newbie using mantine and nextjs 13. I have a problem with the styling.  My folder structure is app, components and pages.  Components and styles (module.css) are in the components folder.  When I run, the Mantine style is only implemented on my home page which is located in the app folder.  Meanwhile, when switching to another page in the pages folder, the mantine style does not appear at all.  Can you help me with what's wrong?

**Comment by ananuness** [comment]
Can you tell if you followed [those steps](https://mantine.dev/guides/next/#setup-with-app-router) to setup mantine in your next 13 project? If yes and the error persists, may you share what exatcly you did so we can find the problem 😊

Also, is your pages folder like `(pages)`?

**Reply by nisamarwa** [reply]
yes, i already followed the instructions. here's my app/layout.jsx:

```
import '@mantine/core/styles.css';

import { MantineProvider, ColorSchemeScript } from '@mantine/core';

export const metadata = {
title: 'StudioGoods',
description: 'I have followed setup instructions carefully',
};

export default function RootLayout({ children }) {
return (





{children}


);
}```

also, here's my structure folder: https://github.com/nisamarwa/StudioGoods/blob/main/app/layout.jsx
